add update policy delay in PGA

limbryan opened the pull request:
Delayed update of the actor policy when updating the models (actor and critic) in PGA

Before update:
<img width="887" alt="image" src="https://user-images.githubusercontent.com/46229436/199570305-e2988ea8-9bc9-47cd-b26d-7f6493200f65.png">

After update:
<img width="911" alt="image" src="https://user-images.githubusercontent.com/46229436/199570428-017178e4-5a6c-414f-b5f1-6d3e2e7921e0.png">

Performance is tested on walker_uni task. Performance seems to improve (albeit only testing on one seed). Coverage relatively unaffected hence improvement mainly in quality, which makes sense given that this is a fix for the actor policy update.

felixchalumeau commented on .pre-commit-config.yaml:
```diff
@@ -24,7 +24,7 @@ repos:
     rev: 0.3.9
     hooks:
       - id: nbstripout
-        args: ["notebooks/"]
```
is this necessary?

felixchalumeau commented:
Looks good!

Two questions:
- is the change in the pre-commit file necessary here?
- did you check the impact on the performances?

limbryan commented:
Thanks!
- Removed the change for the pre-commit (was related to the docs of the CI failing but resolved since merging with the PR that fixes this)
- Added performance test
